arcsi/api/utils.py

```diff
@@ -109,11 +109,10 @@ def get_playlist_existence_and_emptiness(playlist_name):
         playlist_name,
     )
     playlist_exists = az.is_existing_playlist_set_playlist_id()
+    playlist_is_empty = True
```

Comment:
@tuz666 This commit sits on the wrong branch by mistake perhaps I think. Could you please remove this?